### PR TITLE
[relates #20] Ignore ssl errors in proxies

### PR DESCRIPTION
**Pull request by jludvice**

### CODE REVIEW

File: redhat-ipaas-dev-single-tenant.yml [hunk 1]
```diff
@@ -95,6 +95,10 @@ parameters:
   name: POSTGRESQL_VOLUME_CAPACITY
   required: true
   value: 1Gi
+- description: Ignore SSL errors in proxy containers
+  displayName: Insecure skip verify
+  name: INSECURE_SKIP_VERIFY
+  value: 'true'
```
Comment (rhuss) [inline redhat-ipaas-dev-single-tenant.yml]: What is the default of 'token-rp' ? I would love to stick to this default here, too (probably 'false') and switch it on if required.

Comment (dsimansk) [reply redhat-ipaas-dev-single-tenant.yml]: You're right, `false` is default. We can keep those test related params to default "restricted" values and configure them accordingly prior to test execution. 
@jludvice could you pls change this?

Comment (jludvice) [reply redhat-ipaas-dev-single-tenant.yml]: Updated by  	2462607

**Comment by rhuss** [comment]
Merged in the option with #35

**Pull request closed without merging**